src/clientAuth/bearerBody.js

```diff
@@ -23,24 +28,24 @@ class BearerBody extends ClientAuthnMethod {
     if (Object.keys(cis).indexOf('access_token') !== -1) {
```

Comment:
You can just do:

`if (!cis.access_token)`

Reply:
Fixed. Should be cis.access_token instead of !cis.access_token